lib/pf/node.pm

```diff
@@ -296,7 +296,9 @@ sub node_db_prepare {
     ];
 
     $node_statements->{'node_expire_unreg_field_sql'} = get_db_handle()->prepare(
-        qq [ select mac from node where status="reg" and unregdate != 0 and unregdate < now() ]);
```

Comment:
It would be easier, but it may not be the same thing.  

--  
LM


On Tuesday, April 7, 2015 at 15:02 , jrouzierinverse wrote:

> In lib/pf/node.pm (https://github.com/inverse-inc/packetfence/pull/446#discussion_r27909308):
> > @@ -296,7 +296,9 @@ sub node_db_prepare { > ]; > > $node_statements->{'node_expire_unreg_field_sql'} = get_db_handle()->prepare( > - qq [ select mac from node where status="reg" and unregdate != 0 and unregdate < now() ]);  
> would'nt be easier to just say status != 'unreg'
>  
> —
> Reply to this email directly or view it on GitHub (https://github.com/inverse-inc/packetfence/pull/446/files#r27909308).
>  
>  
>  
>  
>